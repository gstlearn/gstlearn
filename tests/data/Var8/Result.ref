Goodness of Fit = 0.21 percent of the Sill
