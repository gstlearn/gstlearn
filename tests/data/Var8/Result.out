
Model characteristics
=====================
Space dimension              = 3
Number of variable(s)        = 1
Number of basic structure(s) = 1
Number of drift function(s)  = 0
Number of drift equation(s)  = 0

Covariance Part
---------------
Spherical
- Sill         =      4.123
- Range        =   1018.983
Total Sill     =      4.123
